### DISCUSSION TITLE
Take the picture sooner?

### DISCUSSION THREAD

**Discussion by Dan-imal** [berrystephenw/OctoText#81] [Ideas]
Don't know if that's possible but lately (since May 5?) my notifications pics are after the table is pushed forward.

It's still a great help so I'm not complaining :)

**Comment by berrystephenw** [comment]
Can you send me your log file? And what settings are you looking for? Print finished?

**Reply by Dan-imal** [reply]
Thank you for the fast reply, I emailed you.  There's really no rush, I was just pointing this out.

Thank you
